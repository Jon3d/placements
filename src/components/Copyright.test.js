import renderer from 'react-test-renderer';
import Copyright from './Copyright';

it('renders Copyright correctly', () => {
  const tree = renderer
    .create(<Copyright />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});