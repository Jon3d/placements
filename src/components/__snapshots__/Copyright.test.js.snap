// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Copyright correctly 1`] = `
<p
  className="MuiTypography-root MuiTypography-body2 MuiTypography-alignCenter css-1vl95ex-MuiTypography-root"
>
  Copyright Â© 
  <a
    className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-xdxgn1-MuiTypography-root-MuiLink-root"
    href="https://mui.com/"
    onBlur={[Function]}
    onFocus={[Function]}
  >
    Your Website
  </a>
   
  2024
  .
</p>
`;
